sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -168,11 +169,11 @@ async def read(
         if populate_quota_info is not None:
             request_options["populateQuotaInfo"] = populate_quota_info
 
-        collection_link = self.container_link
-        self._properties = await self.client_connection.ReadContainer(
-            collection_link, options=request_options, **kwargs
+        coll_link = self.container_link
+        self.client_connection.collection_properties_cache[coll_link] = await self.client_connection.ReadContainer(
+            coll_link, options=request_options, **kwargs
```

Comment:
When I see things like **coll_link** and **container_link**, I immediately think that these are two different things. When in fact, **collection** and **container** are the same.

Reply:
I updated to keep it as consistently as possible as container within the scope of my changes